.github/actions/send-slack-notification-slow-job/action.yml

```diff
@@ -0,0 +1,32 @@
+name: Send a notification for exceeded build time
+description: Send a slack notification if build took more than planned
+
+inputs:
+  max-build-time:
```

Comment:
`max-build-time-seconds` so it became explicit to what we expect here?

Reply:
makes sense, fixed